change/@fluentui-react-positioning-a171cb79-b220-449e-83a3-440d479bdc84.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33260&commitId=64f5676e091120a0067c293a2463bd6bd068b7ae&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1168| Changed|

</details>


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.chromium.png|2| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow.chromium.png|1752| Changed|
|Positioning.Arrow RTL.chromium.png|1727| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.inverted withArrow.chromium.png|8| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|1| Changed|
|Tooltip Converged.withArrow.chromium.png|1| Changed|
|Tooltip Converged.Inverted With Arrow Dark Mode.chromium.png|1| Changed|
|Tooltip Converged.With Arrow High Contrast.chromium.png|18| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>